### DISCUSSION TITLE
Display “tips and tricks” on the bottom menu bar

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#69] [General]
As far as I'm concerned, the “tips and tricks” displayed in the bottom menu bar of Subtitle Workshop 7.02 r3 were a gentle and helpful way to suggest some lesser-known functions of the application to the user. I believe restoring them would be a great added value.

**Comment by chenlung** [comment]
I sometimes see Tips at the bottom (one is about F10). Is it that?

**Comment by URUWorks** [comment]
They are there at startup, but are hidden after a few seconds.

**Comment by GitGianluc** [comment]
Sorry, I didn’t notice! Would it be possible to stick them on the bottom bar, perhaps in the middle, and/or display them in a larger size?

**Reply by URUWorks** [reply]
we can leave them longer.

**Comment by chenlung** [comment]
Do the shortcuts mentioned in them reflect what has been changed by the user? What about OS-specific versions of the shortcuts (e.g. F10)?

**Reply by URUWorks** [reply]
Yes
